test/e2e/storage/BUILD

```diff
@@ -3,6 +3,7 @@ load("@io_bazel_rules_go//go:def.bzl", "go_library")
 go_library(
     name = "go_default_library",
     srcs = [
+        "create.go",
```

Comment:
This go file itself is not e2e test, just a utility.
In addition, `k8s.io/kubernetes/test/e2e/storage/utils` exists already.
So how about moving this code into `k8s.io/kubernetes/test/e2e/storage/utils` ?

Reply:
Thanks for you suggestion, sounds more reasonable.